### DISCUSSION TITLE
Plan to integrate AI.

### DISCUSSION THREAD

**Discussion by 4t8dd** [wezterm/wezterm#5565] [Ideas]
With AI performs great nowaday, is it a good idea to integrate AI to wezterm?

**Comment by wez** [comment]
If you're using the current colloquial definition of AI, then no; it it is a terrible idea to integrate that directly into the terminal.

**Reply by ChristinWhite** [reply]
I like AI, I even like AI for troubleshooting and explaining shell related issues, I unequivocally do not want it built into my terminal. Sure, Warp's AI could be convenient while I was testing it but there's just far too much potential for abuse, security and privacy issues for one of the most highly privileged applications on my system. If generative AI was added to Wezterm I'd be looking for a new terminal and hating every minute of whatever I ended up with that wasn't Wezterm. 

I’m really glad you’ve taken the stance you have, if someone really wants AI there are plenty of ways to integrate AI with CLIs that can be better restricted and are completely optional.

**Comment by ww7** [comment]
Newest ChatGPT for macOS now supporting Terminal.app and iTerm2
It can read a terminal screen and it is useful.
